mlos_bench/mlos_bench/tests/config/schemas/optimizers/test-cases/good/partial/manual_opt_schema_partial_config.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+
+    "class": "mlos_bench.optimizers.ManualOptimizer",
+    "config": {
+        "cycle_tunable_values": []
```

Comment:
This should be rejected, right?  There's nothing to do.

Reply:
With the current logic, if an empty array is provided we first emit a warning for that, and then default to an array with just the default tunable values. Kind of falling back to `OneShotOptimizer` behavior.

I was thinking this could be a sane default, to be overwritten via the globals file with specific values compatible with the current tunable groups. Otherwise, someone copying this default optimizer config would fail immediately.

But if you guys think the latter is better, I can change the behavior to that!

Reply:
I see.  Yeah, that's ok, but let's document it with some comments in the example configs at least.  Thanks!

Reply:
I made an initial attempt at that below, but actually on thinking about it more, I think a better approach is probably something like the following to make it more explicit:

```jsonc
"cycle_tunable_values": [],  // --> error
"cycle_tunable_values": null, // --> error

"cycle_tunable_values": [
  // explicit placeholder for the defaults, so they don't need to be copied:
  // this can be included in the example config and documented as such
  null,
  // alternatively, (if this is easy) - placeholder for defaults (but I think the schema might reject it currently):
  {},
  // explicit values:
  {"tunable_param_1": "tunable_value_1", ... },
],

Reply:
The more I think about it, I actually think `{}` as a placeholder for `default` values is the easiest way to do this rather than `null`.

Let me make a new PR to allow supporting that quickly.

Reply:
@motus does that make sense to you too?

The other alternative I could imagine is `null` as an allowed element in the `tunable_values_cycle` and internally interpret that as `restore_defaults()`.

Reply:
Updated to use the new features in #868!